packages/blade/src/components/BottomSheet/Divider.web.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { makeSize } from '~utils';
+
+const Divider = styled.div(({ theme }) => {
```

Comment:
this seems to have same styles on web and native maybe use `BaseBox` instead? we support `borderBottom` shorthand there so that can be used to set all these properties - https://github.com/razorpay/blade/blob/74c063930c12ddd92f47fc93b2f382dd67382607/packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts#L222

Reply:
So, borderColor isn't supported should we add it?

Reply:
we can do `borderBottom: 2px solid red` kind of thing no?